addon/model/mutators/batched-model-mutator.ts

```diff
@@ -25,10 +26,9 @@ export default class BatchedModelMutator extends ModelMutator<void> {
     this.batch.push(op);
   }
 
-  move(rangeToMove: ModelRange, targetRange: ModelRange) {
-    const op = new MoveOperation(rangeToMove, targetRange);
+  moveToPos(rangeToMove: ModelRange, targetPos: ModelPosition) {
```

Comment:
in all other instances position is written out in full, so would be more consistent to also do that here?

Reply:
I agree!